Documentation/ApiOverview/RequestLifeCycle/RequestAttributes/FrontendPageInformation.rst

```diff
@@ -0,0 +1,43 @@
+..  include:: /Includes.rst.txt
+
+..  index::
+    Request attribute; Frontend page information
+..  _typo3-request-attribute-frontend-page-information:
+
+=========================
+Frontend page information
+=========================
+
+..  versionadded:: 13.0
+    This request attribute replaces various page related properties of
+    :ref:`\\TYPO3\\CMS\\Frontend\\Controller\\TyposcriptFrontendController <tsfe>`.
+
+..  attention::
+    The class is currently still marked as experimental. However, extension
+    authors are encouraged to use information from this request attribute
+    instead of the :php:`TyposcriptFrontendController` properties already: TYPO3
+    Core v13 will try to not break especially the getters / properties not
+    marked as :php:`@internal`.
```

Comment:
I kind of find this sentence very complex to follow but don't really have a better idea. something feels odd.

Reply:
Yup, I assume, this will be adjusted/vanish with time. This was the first patchset and I assume, some others will come to streamline the area.